ui/src/app/pages/setting/component/adaptertype/adaptertype-list/facility-adapter-list.component.html

```diff
@@ -0,0 +1,551 @@
+         <li> <button class="pagination-next" type="button" disabled="{{nextbtnDisabled}}" on-click="next()"></button></li>
+     </ul>
+</clr-dg-footer>
+
+</clr-datagrid>
+
+<clr-wizard #addFacilityAdapterwizard [(clrWizardOpen)]="addFacilityAdapterOpen">
+    <clr-wizard-title>New Facility Adapter</clr-wizard-title>
+
+    <clr-wizard-button [type]="'cancel'">Close</clr-wizard-button>
+    <clr-wizard-button [type]="'previous'">Previous</clr-wizard-button>
+    <clr-wizard-button [type]="'next'">Next</clr-wizard-button>
+    <clr-wizard-button [type]="'finish'" >Done</clr-wizard-button>
+
+    <clr-wizard-page [clrWizardPageNextDisabled]="!formPageOne.valid">
+        <ng-template clrPageTitle>Create</ng-template> 
+        <form #formPageOne="ngForm">
+            <section class="form-block">
+                <div class="form-group ">
+                    <label class="required" for="gForm_5">Type</label>
+                        <div class="select" style="width:184px; ">
+                            <select id="gForm_5" [ngModelOptions]="{standalone: true}"  [ngModel]="newFacilityAdapter.type" (ngModelChange)="newFacilityAdapter.type=$event"> 
+                                    <option value="OtherCMDB">CMDB</option>
+                                    <option value="OtherDCIM">DCIM</option>
+                            </select>
+                        </div>
+                </div>
+                <div class="form-group">
+                    <label class="required">Display Name</label>
+                    <label for="nameInput" aria-haspopup="true" role="tooltip"
+                        [class.invalid]="name.touched && !name.valid"
+                        class="tooltip tooltip-validation tooltip-md tooltip-bottom-right">
+                        <input type="text" id="nameInput" required
+                        pattern="^[a-zA-Z][a-zA-Z0-9_]*$"
+                        [(ngModel)]="newFacilityAdapter.displayName" name="name" #name="ngModel">
+                        <span class="tooltip-content">This field cannot be empty! Only letters and numbers can be entered and start with a letter</span>
+                    </label>
+                </div>
+                <div class="form-group">
+                    <label for="favInput">Description</label>
+                    <textarea clrTextarea [(ngModel)]="newFacilityAdapter.description" name="description" ></textarea>
+                </div>
+            </section>
+        </form>
+    </clr-wizard-page>
+
+    <clr-wizard-page [clrWizardPageNextDisabled]="!formPageOne.valid">
+        <ng-template clrPageTitle>Config adapter</ng-template> 
+        <ng-template clrPageNavTitle>
+            Config
+        </ng-template> 
+        <clr-alert [clrAlertType]="'alert-info'" [clrAlertClosable]="false">
+            <div class="alert-item">
+                <ul>
+                    <li style="width: max-content;">This is a commands list contains two predefined commands.</li>
```

Comment:
Flowgate will trigger the commands list follow for each adapter with the specified trigger cycle. The syncmetadata command is planed to sync the asset metadata information. The syncmetricsdata command is planed to sync the metrics data. You can always add more commands when needed.